modules/builds-configmaps-use-case-setting-command-line-arguments.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+//* builds/builds-configmaps.adoc
+
+[id="builds-configmaps-use-case-setting-command-line-arguments_{context}"]
+= Setting command-line arguments with ConfigMaps
+
+A ConfigMap can also be used to set the value of the command or arguments in a container. This is accomplished using the Kubernetes substitution syntax `$(VAR_NAME)`. Consider the following ConfigMaps:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: special-config
+  namespace: default
+data:
+  special.how: very
+  special.type: charm
+----
+
+.Procedure
+
+* To inject values into the command line, you must consume the keys you want to use as environment variables, as in the Consuming in Environment Variables use case. Then you can refer to them in a container's command using the `$(VAR_NAME)` syntax.
++
+.Sample pod specification configured to inject specific environment variables
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dapi-test-pod
+spec:
+  containers:
+    - name: test-container
+      image: gcr.io/google_containers/busybox
+      command: [ "/bin/sh", "-c", "echo $(SPECIAL_LEVEL_KEY) $(SPECIAL_TYPE_KEY)" ]
```

Comment:
I think it would be useful to have a call out for this line, highlighting what is happening here. I think this would help draw the reader's eye to the salient element of the code block.